app/containers/RegisterEmail/actions.js

```diff
@@ -83,6 +83,15 @@ export function resetState() {
   };
 }
 
+function ValidateEmail(mail)   
+{  
+ if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(myForm.emailAddr.value))  
+  {  
+    return (true)  
+  }  
+    alert("You have entered an invalid email address!")  
```

Comment:
you could put the `alert()` function after receiving the return value of this function in the view. that'd be neater